### PR TITLE
Bugfix FXIOS-5862 [v121] Remove SwiftyJSON from TokenServerToken.swift

### PR DESCRIPTION
**Pull request by imsanchit**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-5862)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/13380)

## :bulb: Description
Remove SwiftyJSON from TokenServerToken.swift

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [x] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: Account/TokenServerClient.swift [hunk 1]
```diff
@@ -52,8 +52,8 @@ public struct TokenServerToken {
         return nil
     }
 
-    public func asJSON() -> JSON {
-        let D: [String: AnyObject] = [
+    public func asJSON() -> [String: Any] {
+        let D: [String: Any] = [
```
Comment (lmarceau) [inline Account/TokenServerClient.swift:56]: nit
```suggestion
        let dictionnary: [String: Any] = [
```

File: Account/TokenServerClient.swift [hunk 2]
```diff
@@ -62,7 +62,7 @@ public struct TokenServerToken {
             "duration": NSNumber(value: durationInSeconds as UInt64),
             "remoteTimestamp": NSNumber(value: remoteTimestamp),
         ]
-        return JSON(D as NSDictionary)
+        return D
```
Comment (lmarceau) [inline Account/TokenServerClient.swift:65]: nit
```suggestion
        return dictionnary
```

File: Account/TokenServerClient.swift [hunk 3]
```diff
@@ -31,15 +31,15 @@ public struct TokenServerToken {
                self.api_endpoint == other.api_endpoint
     }
 
-    public static func fromJSON(_ json: JSON) -> TokenServerToken? {
+    public static func fromJSON(_ jsonDict: [String: Any]) -> TokenServerToken? {
```
Comment (lmarceau) [inline Account/TokenServerClient.swift:34]: Could you write unit tests for the `fromJSON` and for the `asJSON` methods?

Comment (imsanchit) [reply Account/TokenServerClient.swift:34]: @lmarceau updated with unit tests for both methods, please have a look.

**Review by lmarceau — APPROVED** [review]

**Pull request merged**